Project-15-place-picker/src/components/Places/Places.module.scss

```diff
@@ -0,0 +1,95 @@
+.places-category {
+  max-width: 85rem;
+  margin: 2rem auto;
+  padding: 1rem;
+  border: 2px solid #0d373e;
+  border-radius: 8px;
+
+  h2 {
+    font-family: "Raleway", sans-serif;
```

Comment:
@sknepali  `font-family: "Raleway", sans-serif;` is used multiple times. Consider making this a variable or a rule associated with the root element, or a shared class to boost reusability.